src/lib.rs

```diff
@@ -650,21 +655,28 @@ impl Signer for ExternalAccountKey {
     }
 }
 
-fn nonce_from_response(rsp: &Response<Body>) -> Option<String> {
+fn nonce_from_response(rsp: &Response<Incoming>) -> Option<String> {
     rsp.headers()
         .get(REPLAY_NONCE)
         .and_then(|hv| String::from_utf8(hv.as_ref().to_vec()).ok())
 }
 
 #[cfg(feature = "hyper-rustls")]
-struct DefaultClient(hyper::Client<hyper_rustls::HttpsConnector<HttpConnector>>);
+struct DefaultClient(
+    hyper_util::client::legacy::Client<hyper_rustls::HttpsConnector<HttpConnector>, Full<Bytes>>,
```

Comment:
Can we import `hyper_util::client::legacy::Client as LegacyClient`?

Reply:
I made it `HyperClient` let me know if that's okay!

Reply:
Yes, that's better!